web/routing.go

```diff
@@ -136,6 +140,7 @@ func SetupRoutes(router *echo.Echo) error {
 				DefaultContentTypeOffer: echo.MIMETextHTML,
 			}),
 			middlewares.CheckIE,
+			middlewares.CheckInstanceBlocked,
```

Comment:
What will be shown to the user when their instance is blocked?

Reply:
The user will see the default HTTP error page with a `StatusPaymentRequired` HTTP code sent by the middleware, which is like this :

![blocked](https://user-images.githubusercontent.com/43468959/51688685-65fc6880-1ff5-11e9-8181-00e608711425.png)

A better-looking page is planned for the future